### ISSUE TITLE
MySQL does not start

### ISSUE THREAD

**Issue by vikasprogrammer** [cscart/server-ansible-playbooks#10]
I used this

`cd ~/playbooks/ && ansible-playbook -e @config/main.json -c local -i inventory_php7 lemp7.yml
`
to install on Ubuntu 16.04, but the MySQL server does not start.

**Comment by cristihcd** [comment]
Edit `~/playbooks/roles/mysql/common/templates/my.cnf.j2` and comment out the following lines:

```
;pid             = {{ mysql.pid|default("/var/run/mysqld/mysqld.pid") }}
;key_buffer            = {{ mysql.key_buffer|default("64M") }}
;myisam-recover        = BACKUP
```

then retry.

**Comment by cristihcd** [comment]
also see the fork (readme and my.cnf.j2): https://github.com/cristihcd/server-ansible-playbooks

**Comment by Protopopys** [comment]
Fixed and update Mysql to 5.7.
